BasicPrelude.hs

```diff
@@ -250,5 +274,24 @@ fpToString = id
 decodeUtf8 :: ByteString -> Text
 decodeUtf8 = decodeUtf8With lenientDecode
 
+getLine :: MonadIO m => m Text
```

Comment:
Can you add some `@since` comments to the newly exposed identifiers?

Reply:
Do you mean newly exported functions (`traverse_`, ...) or  functions that are redefined (`readIO`, `maximumBy`, ...)?

Reply:
Ideally: anything which is exposed now but wasn't exposed before would have a `@since`. Unfortunately, this can't be done for identifiers reexported from other modules, so we can't do anything about them.